### PR TITLE
Overlay for Kauai w/ tooltip

### PR DESCRIPTION
**Pull request by kmal808**
Kauai County: Adding a temporary overlay with an "Under Construction" tooltip until the zoning data can be sus'd out of the parcels data. 
SCR-20221129-v8v

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -663,6 +663,18 @@ <h2 class="f5 mb1 black-40">Overlays</h2>
 						Federal/State Public Lands
 					</label>
 
+					<br />
```
Comment (tyliec) [inline index.html]: We probably don't need this part, as Kauai will be forever turned on anyway :)

File: scripts/map.js [hunk 2]
```diff
@@ -100,6 +100,7 @@ var loadZones = function (geojson) {
 
 	// Turn on federal/state land by default
 	$('input[name="Overlay"][value="fs"]').prop('checked', true)
+	$('input[name="Overlay"][value="kauai"]').prop('checked', true)
```
Comment (tyliec) [inline scripts/map.js]: Can remove this as well

Comment (tyliec) [reply scripts/map.js]: You can probably override adding Kauai with :

```javascript
overlays["kauai"].addTo(map)
```

**Review by tyliec — CHANGES_REQUESTED** [review]
We don't need to have a checkbox in HTML since we want this turned on with no toggle option

File: scripts/map.js (hunk 2 above)
Comment (kmal808) [reply scripts/map.js]: Instead of the hidden checkbox, got it.

**Comment by kmal808** [comment]
~~The checkbox should be hidden, is it not?~~ I used the hidden(or not lol) checkbox as it was the easiest (for me) way that I knew how to get it up and going quick. ~~Will change tonight~~(_changed_). Appreciate it. 

Edit: This was super helpful. Seriously appreciate it 🙏

**Review by tyliec — APPROVED** [review]

**Pull request merged**